src/Workleap.Extensions.Mongo.Analyzers/IndexAttributeUsageAnalyzer.cs

```diff
@@ -32,18 +32,33 @@ public override void Initialize(AnalysisContext context)
     private static void OnCompilationStarted(CompilationStartAnalysisContext context)
     {
         var analyzer = new IndexAttributeUsageAnalyzerImplementation(context.Compilation);
-        if (analyzer.IsValid)
+        if (analyzer.ShouldScan)
```

Comment:
If you want to change this, then please change it also in our other analyzers like mediatr so we can keep them standard